.circleci/config.yml

```
@@ -25,7 +25,7 @@ commands:
     steps:
       - node-install
       - restore_cache:
-          key: dependency-cache-ruby-2-7-{{ checksum "package-lock.json" }}-{{ checksum "Gemfile.lock" }}
+          key: dependency-cache-ruby-3-1-{{ checksum "package-lock.json" }}-{{ checksum "Gemfile.lock" }}
```

Maybe this should checksum `.ruby-version` so we don't have to maintain it manually.

```suggestion
          key: dependency-cache-ruby-{{ checksum ".ruby-version" }}-{{ checksum "package-lock.json" }}-{{ checksum "Gemfile.lock" }}
```